### PR TITLE
FEAT: fixed multi_desktop error

### PR DESCRIPTION
**Pull request by maxcapodi78**
added import_3d_model and export_3d_model to both modeler and Analysis3D.py

### CODE REVIEW

**Review by Samuelopez-ansys — APPROVED** [review]

File: pyaedt/application/Analysis3D.py [hunk 1]
```diff
@@ -504,6 +504,94 @@ def copy_solid_bodies_from(self, design, assignment=None, vacuum=True, pec=True,
...
+        Parameters
+        ----------
+        input_file : str
+            Full path and name of the CAD file.
+        healing : bool, optional
+            Whether to perform healing. The default is ``False``, in which
+            case healing is not performed.
+        healing : int, optional
+            Whether to perform healing. The default is ``0``, in which
+            case healing is not performed.
```
Comment (PipKat) [inline pyaedt/application/Analysis3D.py:537]:
```suggestion
        healing : bool, optional
            Whether to perform healing. The default is ``False``.
```
Duplicate entry removed (as well as unnecessary text).

File: pyaedt/application/Analysis3D.py [hunk 2]
```diff
@@ -504,6 +504,94 @@ def copy_solid_bodies_from(self, design, assignment=None, vacuum=True, pec=True,
...
+        create_lightweigth_part : bool ,optional
+            Either to import lightweight or not.
+        group_by_assembly : bool, optional
+            Either import by sub-assembly or individual parts. The default is ``False``.
+        create_group : bool, optional
+            Either to create a new group of imported objects. The default is ``True``.
+        separate_disjoints_lumped_object : bool, optional
+            Either to automatically separate disjoint parts. The default is ``False``.
+        import_free_surfaces : bool, optional
+            Either to import free surfaces parts. The default is ``False``.
```
Comment (PipKat) [inline pyaedt/application/Analysis3D.py:553]:
```suggestion
            Whether to import material names from the file if present. The
            default is ``False``.
        create_lightweigth_part : bool ,optional
            Whether to import a lightweight part. The default is ``True``.
        group_by_assembly : bool, optional
            Whether to import by subassembly. The default is ``False``, in which
            case the import is by individual parts.
        create_group : bool, optional
            Whether to create a group of imported objects. The default is ``True``.
        separate_disjoints_lumped_object : bool, optional
            Whether to automatically separate disjoint parts. The default is ``False``.
        import_free_surfaces : bool, optional
            Whether to import free surfaces parts. The default is ``False``.
```

File: pyaedt/application/Analysis3D.py [hunk 3]
```diff
@@ -504,6 +504,94 @@ def copy_solid_bodies_from(self, design, assignment=None, vacuum=True, pec=True,
...
+        heal_stl : bool, optional
+            Whether to heal the stl file on import or not. Default is ``True``.
+        reduce_stl : bool, optional
+            Whether to reduce the stl file on import or not. Default is ``True``.
+        reduce_percentage : int, optional
+            Stl reduce percentage. Default is  ``0``.
+        reduce_error : int, optional
+            Stl error percentage during reduce operation. Default is  ``0``.
+        merge_planar_faces : bool, optional
+            Stl automatic planar face merge during import. Default is ``True``.
```
Comment (PipKat) [inline pyaedt/application/Analysis3D.py:565]:
```suggestion
            Tolerance on the point. The default is ``1e-6``.
        heal_stl : bool, optional
            Whether to heal the STL file on import. The default is ``True``.
        reduce_stl : bool, optional
            Whether to reduce the STL file on import. The default is ``True``.
        reduce_percentage : int, optional
            Percentage to reduce the STL file by if ``reduce_stl=True``. The default is ``0``.
        reduce_error : int, optional
            Error percentage during STL reduction operation. The default is ``0``.
        merge_planar_faces : bool, optional
            Whether to merge planar faces during import. The default is ``True``.
```

File: pyaedt/modeler/cad/Primitives.py [hunk 4]
```diff
@@ -4607,6 +4607,123 @@ def get_vertices_of_line(self, assignment):
...
+        file_format : str, optional
+            Format of the file. The default is ``".step"``.
+        assignment_to_export : list, optional
+            List of objects to export. The default is ``None``.
+        assignment_to_remove : list, optional
+            List of objects to remove. The default is ``None``.
+        major_version : int, optional
+            File format major version. Default is -1.
+        minor_version : int, optional
+            File format major version. Default is -1.
```
Comment (PipKat) [inline pyaedt/modeler/cad/Primitives.py:4639]:
```suggestion
            File format major version. The default is -1.
        minor_version : int, optional
            File format major version. The default is -1.
```
Can these last two descriptions be clearer?

**Review by Samuelopez-ansys — APPROVED** [review]

**Pull request merged**